test/cases/channel/close.test.ts

```diff
@@ -0,0 +1,147 @@
+import setupTestEnvironment from '../../../src';
+import mediated from '../../../topologies/mediated.json';
+import { tokenAddresses, toWei } from '../../../src/util/token';
+import { LuminoTestEnvironment } from '../../../src/types/lumino-test-environment';
+import { Dictionary } from '../../../src/util/collection';
+import { LuminoNode } from '../../../src/types/node';
+import { verifyChannel } from '../../utils';
+import {
+    SETUP_TIMEOUT,
+    TEARDOWN_TIMEOUT,
+    ChannelState,
+    State,
+    TEST_TIMEOUT,
+} from '../../common';
+
+interface closeParams {
+    token: string;
+    partner: string;
+}
```

Comment:
Maybe we could change ths for:
```suggestion
interface closeParams {
    tokenAddress: string;
    partnerAddress: string;
}
```
and add that interface to lumino-js-sdk.d.ts so we can call closeChannel(params) more easily

Reply:
> Maybe we could change ths for:
> 
> and add that interface to lumino-js-sdk.d.ts so we can call closeChannel(params) more easily

i like this idea. it could both simplify the SDK calls as well as make the tests cleaner.

if @JONAF2103 agrees, i'll create an issue. we'll have to think about how best to design these "params" classes or interfaces so that they make sense across all the SDK operations.

Reply:
created issue [RLP-1090](https://jirainfuy.atlassian.net/browse/RLP-1090) for this